dashboard/app/views/shared/_user_header.html.haml

```diff
@@ -9,7 +9,7 @@
       &#x25BC;
     %div{style: 'position: relative; top: 3px;'}
       .user_options{style: 'display: none;'}
-        - if current_user.teacher?
+        - if current_user.teacher? && !request.cookies['pm'] == 'new_header'
```

Comment:
I wonder if it's worth adding a helper method for this somewhere. Not too sure where the right somewhere is.